app/views/errors/not_found.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1><%= t 'errors.not_found' %></h1>
+<p>If you'd like to let us know there's been a problem, please <a href="mailto:me@email.com">email the team</a>.</p>
```

Comment:
This requires copy. It's the message which appears below the 404 page title. @sampling